RecoMuon/TrackerSeedGenerator/interface/SeedMvaEstimatorPhase2.h

```diff
@@ -0,0 +1,94 @@
+#ifndef __PhysicsTools_PatAlgos_SeedMvaEstimator__
+#define __PhysicsTools_PatAlgos_SeedMvaEstimator__
```

Comment:
this does not conform with the standard, see rule 4.1: http://cms-sw.github.io/cms_coding_rules.html#4--technical-coding-rules-1